package.json

```diff
@@ -18,10 +18,12 @@
         "build-runtime": "webpack --config webpack.runtime.js",
         "build-function": "webpack --config webpack.function.js",
         "test": "node node_modules/mocha/bin/_mocha -r mocha.js src/**/*.spec.ts tests/e2e/**/*.spec.ts --timeout 30000",
+        "test-e2e": "node node_modules/mocha/bin/_mocha -r mocha.js tests/e2e/**/*.spec.ts --timeout 3000000",
```

Comment:
@f-alizada you might want to sync up with @JMach1 
he is running a specific test folder for e2e. And your branch is running all tests, we are double running tests.

Reply:
This PR is on hold currently, and from change we can see that for e2e tests we have separate folder so both pr's running different tests

Reply:
hmm yes, it is under different folder than src, okay it is cool here then